basic/atm/ATM_machine.py

```diff
@@ -0,0 +1,93 @@
+from Costumer import Costumer
+
+PATH = r"C:\Users\Zolberg\Desktop\python\course\Targilim\ATM\Costumers'_DB.txt"
+COSTUMER_DB = open(PATH, 'r')
```

Comment:
I will move it to a var, but why? It should be fixed.

Reply:
Because file descriptors should be temporary object that you should hold only for the purpose of interacting with the file. by putting in a const you are occupying the file for the entire runtime of the program